tools/src/h5dump/h5dump.h

```diff
@@ -12,6 +12,181 @@
+ * \li <strong>--packedbits=L</strong> Print packed bits as unsigned integers, using mask
+ *                          format L for an integer dataset specified with
+ *                          option -d. L is a list of offset,length values,
+ *                          separated by commas. Offset is the beginning bit in
+ *                          the data value and length is the number of bits of
+ *                          the mask.
+ * \li <strong>--region</strong>       Print dataset pointed by region references
+ *
+ * \subsection subsec_cltools_h5dump_options_fmt Formatting Options
+ * \li <strong>--escape</strong>       Escape non printing characters
+ * \li <strong>--string</strong>       Print 1-byte integer datasets as ASCII
+ * \li <strong>--noindex</strong>      Do not print array indices with the data
+ * \li <strong>--format=T</strong>     Set the floating point output format
+ * \li <strong>--sort_by=Q</strong>    Sort groups and attributes by index Q
+ * \li <strong>--sort_order=Z</strong> Sort groups and attributes by order Z
+ * \li <strong>--no-compact-subset</strong>  Disable compact form of subsetting and allow the use
+ *                                       of "[" in dataset names.
+ * \li <strong>--width=N</strong>      Set the number of columns of output. A value of 0 (zero)
+ *                                       sets the number of columns to the maximum (65535).
+ *                                       Default width is 80 columns.
+ *
+ * \subsection subsec_cltools_h5dump_options_xml XML Options
+ * \li <strong>--xml</strong>          Output in XML using Schema
+ * \li <strong>--use-dtd</strong>      Output in XML using DTD
+ * \li <strong>--xml-dtd=U</strong>    Use the DTD or schema at U
+ * \li <strong>--xml-ns=S</strong>     (XML Schema) Use qualified names n the XML
+ *                                       ":": no namespace, default: "hdf5:"
+ *                                       E.g., to dump a file called "-f", use h5dump -- -f
+ *
+ * \subsection subsec_cltools_h5dump_options_subset Subsetting Options
+ * Subsetting is available by using the following options with a dataset
+ * option. Subsetting is done by selecting a hyperslab from the data.
+ * Thus, the options mirror those for performing a hyperslab selection.<br />
+ * One of the \b START, \b COUNT, \b STRIDE, or \b BLOCK parameters are mandatory if you do subsetting.
+ * The \b STRIDE, \b COUNT, and \b BLOCK parameters are optional and will default to 1 in
+ * each dimension. \b START is optional and will default to 0 in each dimension.
+ *
+ * \li <strong>--start=START</strong>    Offset of start of subsetting selection
+ * \li <strong>--stride=STRIDE</strong>  Hyperslab stride
+ * \li <strong>--count=COUNT</strong>    Number of blocks to include in selection
+ * \li <strong>--block=BLOCK</strong>    Size of block in hyperslab
+ *  \b START, \b COUNT, \b STRIDE, and \b BLOCK - is a list of integers the number of which are equal to the
```

Comment:
I think (I could be wrong) that each of START, COUNT, STRIDE and BLOCK is a list of integers, and as written it could be interpreted that START, COUNT, STRIDE and BLOCK is a list.